app/views/AlreadyRegisteredView.scala.html

```diff
@@ -34,6 +34,6 @@ <h1 class="govuk-heading-m">@messages("alreadyRegistered.heading.title")</h1>
      classes = "govuk-!-margin-top-0"
     ))
 
-    <p class="govuk-body" id="account-redirect">@messages("alreadyRegistered.account.goto") <a id="account-link" href="@routes.IndexController.onPageLoad().url" >@messages("alreadyRegistered.account.link.text")</a></p>
+    <p class="govuk-body" id="account-redirect">@messages("alreadyRegistered.account.goto") <a id="account-link" href=@accountLink>@messages("alreadyRegistered.account.link.text")</a></p>
```

Comment:
then can call config.accountFrontendHomeUrl here.

Reply:
Should be done